demos/guru_scripts/docker_notebook/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:16.04
+
+ENV DEV_VERSION 2.6.1
+RUN useradd -ms /bin/bash tigergraph
+
+RUN apt-get -qq update && apt-get install -y --no-install-recommends curl iproute2 net-tools cron ntp locales vim emacs wget git tar unzip jq uuid-runtime openssh-client openssh-server > /dev/null && \
+  mkdir /var/run/sshd && \
+  echo 'root:root' | chpasswd && \
+  echo 'tigergraph:tigergraph' | chpasswd && \
+  sed -i 's/PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config && \
+  sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd && \
+  curl -s -L http://dl.tigergraph.com/developer-edition/tigergraph-2.6.1-developer.tar.gz \
```

Comment:
@yaniv256
 can you add comments above this line, to point to the latest version?
curl -s -L http://dl.tigergraph.com/developer-edition/tigergraph-2.6.1-developer.tar.gz

Reply:
curl -s -OL https://dl.tigergraph.com/tigergraph-developer-latest.tar.gz